arm-customer-insights/2017-01-01/swagger/customer-insights.json

```diff
@@ -3937,6 +4036,7 @@
         "calculationWindow": {
           "type": "string",
           "enum": [
+            "Lifetime",
```

Comment:
This is actually a bug, Lifetime was missing in previous swagger. No service side change about this.